tools/wasi/uvwasi.hpp

```diff
@@ -0,0 +1,36 @@
+// Fizzy: A fast WebAssembly interpreter
+// Copyright 2021 The Fizzy Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include <wasi_types.h>
+#include <memory>
+
+namespace fizzy::wasi
+{
+/// UVWASI interface.
+class UVWASI
+{
+public:
+    virtual ~UVWASI();
```

Comment:
` = default`.

Reply:
This produces `error: 'UVWASI' has no out-of-line virtual method definitions; its vtable will be emitted in every translation unit [clang-diagnostic-weak-vtables,-warnings-as-errors]`

https://app.circleci.com/pipelines/github/wasmx/fizzy/6667/workflows/39a77e98-002f-42e8-a916-19acec64fcbc/jobs/58758/parallel-runs/0/steps/0-111